sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/postgreSQL/StringUtils.scala

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
nit: Since we already have the dir named `pgSQL` in `sql/core/src/test/resources/sql-tests/inputs/pgSQL`, `postgreSQL` -> `pgSQL`? Both names is ok, but I like a consistent name.

Reply:
How about a follow-up to change `pgSQL` to `postgreSQL`. I prefer the official full name.

Reply:
Yea, ok to me.